graide.spec

```diff
@@ -34,17 +32,15 @@ if sys.platform == 'win32' :
     pathex.append(pth)
     ext = '.exe'
 
-a = Analysis(['build/scripts-2.7/graide'],
+a = Analysis(['graide'],
              pathex=pathex,
              hiddenimports=[],
              hookspath=None)
 pyz = PYZ(a.pure)
 bins = a.binaries
+for d in glob.glob('lib/graide/grcompiler/*.*') :
+    bins += [(d[22:], d, 'BINARY')]
 if sys.platform == 'win32' :
-    for d in ('zlib1', 'freetype6', 'graphite2', 'msvcr100') :
-        bins += [(d + '.dll', 'build/scripts-2.7/' + d + '.dll', 'BINARY')]
-    for d in glob.glob('grcompiler_win/*.*') :
-        bins += [(d[15:], d, 'BINARY')]
```

Comment:
If not here, where do these dependencies get pulled in? (Likewise for Darwin)

Reply:
In setup.py there is install_requires which makes setuptools pull them in if they are released on PyPI which is the case for all dependencies except graphite2. If you merge silnrsi/graphite#37 I volunteer to publish the Python package. The setuptools will install binary wheels if there are any for the platform in question. All dependencies' Python packages that depend on dynamic libraries have binary wheels available for Windows, macOS and Linux. Those are [freetype-py](https://pypi.org/project/freetype-py/#files) and [PySide2](https://pypi.org/project/PySide2/#files); [PySide](https://pypi.org/project/PySide/#files) only has Windows binaries.

I added grcompiler.py, enabling setuptools to download and compile grcompiler for each supported platform with GRCOMPILER_BUNDLE_VERSION environment variable set. Only the person publishing the graide package on PyPI would have to use that.